ExpantaNum.js

```diff
@@ -414,7 +443,7 @@
   P.reciprocate=P.rec=function (){
     if (ExpantaNum.debug>=ExpantaNum.NORMAL) console.log(this+"^-1");
     if (this.isNaN()||this.eq(ExpantaNum.ZERO)) return ExpantaNum.NaN.clone();
-    if (this.abs().gt("2e323")) return ExpantaNum.ZERO.clone();
+    if (this.abs().gt("2e323")) return ExpantaNum.ONE.clone().div(this);
```

Comment:
While handling this rage would be required if the library needs to support very small numbers, that is not the case right now.